js/lib/Country.js

```diff
@@ -108,45 +108,45 @@ var Country = function (options) {
 /**
  * Return an array of the ids for all ISO 3166-1 alpha-2 code that
  * this copy of ilib knows about.
- * 
+ *
  * @static
  * @return {Object} an object of country code that this copy of ilib knows about.
  */
 Country.getAvailableCode = function() {
 	var ret = [],
-		country,
+		code,
 		countries = new ResBundle({
-			name: "ctrynames"
+			name: "ctryreverse"
```

Comment:
Not sure why we have to load in the same data as a ResBundle here when the data is already loaded on line 92.

Reply:
`getAvailableCode` is static function, so it need to load data. 
Is it better to save data in `ilib.data`?